developer/src/kmcmplib/src/CheckFilenameConsistency.cpp

```diff
@@ -1,13 +1,16 @@
 
+#define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING 1
 #include "pch.h"
-
 #include "compfile.h"
 #include <comperr.h>
 #include "kmcmpdll.h"
 #include <io.h>
 #include <string>
 #include "CheckFilenameConsistency.h"
 #include "kmx_u16.h"
+#include <experimental/filesystem>
+namespace fs = std::experimental::filesystem;
+using std::experimental::filesystem::directory_iterator;
```

Comment:
C++17, is that OK?

Reply:
windows/src/engine/testhost/testhost.cpp just uses `#include <filesystem>`

Reply:
```suggestion
#include <filesystem>
namespace fs = std::filesystem;
using fs::directory_iterator;
```

Reply:
I wanted to do exactly your solution but this was not possible therefore I changed it to experimental/... and set the DEPRECATION_WARNING